### PR TITLE
[doc] update ray image dependencies

### PR DESCRIPTION
**Pull request by aslonnie**
update deps after 2.11 release, per latest nightly build from commit https://github.com/ray-project/ray/commit/a1303138acce2ff43ae710777ab212c6b93f49cb

### CODE REVIEW

File: doc/source/ray-overview/installation.rst [hunk 1]
```diff
@@ -427,14 +427,14 @@ Docker images for Python 3.9.
     .. tab-item:: ray (Python 3.9)
         :sync: ray (Python 3.9)
 
-        Ray version: nightly (`e1d7025 <https://github.com/ray-project/ray/commit/e1d7025b4eb30911739681f33586be01f7dcbf5b>`_)
+        Ray version: nightly (`2eb4a81 <https://github.com/ray-project/ray/commit/2eb4a8119f903f79b78c01eaa9db06c6c390051c>`_)
```
Comment (aslonnie) [inline doc/source/ray-overview/installation.rst:430]: @khluu  this is not really from nightly..

**Comment by khluu** [comment]
This PR should use latest build on master & its artifacts

File: doc/source/ray-overview/installation.rst (hunk 1 above)
Comment (khluu) [reply doc/source/ray-overview/installation.rst:430]: we can rename `nightly` with its SHA I think?

Comment (khluu) [reply doc/source/ray-overview/installation.rst:430]: this is just a doc section listing all dependencies up until the last time it's updated, so I think we can rename it as long as it's easy for user to understand

Comment (aslonnie) [reply doc/source/ray-overview/installation.rst]: picking from latest nightly build:
https://buildkite.com/ray-project/postmerge/builds/4047

**Review by khluu — APPROVED** [review]

**Pull request merged**